build/devenv/tests/e2e/smoke_test.go

```diff
@@ -210,16 +218,52 @@ func TestE2ESmoke(t *testing.T) {
 				// 	https://smartcontract-it.atlassian.net/browse/CCIP-7351
 				expectFail: false,
 			},
+			{
+				name:        "src_dst msg execution with EOA receiver and token transfer",
+				srcSelector: selectors[0],
+				dstSelector: selectors[1],
+				finality:    1,
+				receiver:    mustGetEOAReceiverAddress(t, c, selectors[1]),
+				mandatoryCCVs: []protocol.CCV{
+					{
+						CCVAddress: getContractAddress(t, in, selectors[0], datastore.ContractType(committee_verifier.ContractType), committee_verifier.Deploy.Version(), "", "committee verifier"),
```

Comment:
I think this should be `committee_verifier.ProxyType` and use the const `ccvEvm.DefaultCommitteeVerifierQualifier` to get the right address